scripts/package/postinstall.sh

```diff
@@ -43,6 +43,7 @@ Type=simple
 User=observiq-otel-collector
 Group=observiq-otel-collector
 Environment=PATH=/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin
+Environment=OIQ_OTEL_COLLECTOR_STORAGE=/opt/observiq-otel-collector/storage
```

Comment:
Can we also add this env var to Windows?

Reply:
Would be nice, but I am not sure what that change would look like for wix.

Reply:
Done